modules/EmailTemplates/EditViewMain.html

```diff
@@ -188,6 +188,18 @@
 						</td>
 					</tr>
 					<tr>
+						<td width="15%" scope="row">
+							{MOD.LBL_WIDTH}
+						</td>
+						<td>
+							<div class="panel-toolbar">
+								<div class="template-container">
+									<input type="number" class="form-control input-sm" id="mozaik_width_set" onkeyup="mozaik.setWidth(parseInt(this.value)+'px');" onmouseup="mozaik.setWidth(parseInt(this.value)+'px'); "value="{template_width}" >
```

Comment:
type="number"?

In SuiteP, the style is incorrect. It should have a similar style to the text input fields.